docs/plugins/dotnet-plugin-events/create-entity-controller.md

```diff
@@ -48,24 +48,24 @@ Example:
 async afterCreateEntityController(
   context: DsgContext,
   eventParams: CreateEntityControllerParams,
-  modules: ModuleMap
+  files: FileMap<F>
 ) {
   const { entity, resourceName, apisDir } = eventParams;
   const controllerPath = join(apisDir, `${resourceName}Controller.cs`);
-  const controllerFile = modules.get(controllerPath);
+  const controllerFile = files.get(controllerPath);
 
   if (controllerFile) {
     const updatedCode = controllerFile.code.replace(
```

Comment:
This (.code.replace) is only relevant for Nodejs where the ".code" is a string. In the Dotnet plugins, the ".code" is an ASTNode and the behaviour will be different. Please talk to Paz to consult on what we can show (this will be true for many of those dotnet examples)
@PazYanoverr @dericksozo

Reply:
Thanks for the review, Muly! Paz, I'll keep posted for next steps.